tests/scripts/test_generate_fits.py

```diff
@@ -0,0 +1,32 @@
+import os
+import tempfile
+
+from generate_fits import PosteriorsGenerator
+
+
+def test_generate_fits():
+    """Tests posteriors generation for a variety of
+    samplers using a fixed seed."""
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        for sampler in [
+            "dynesty",
+            "svi",
+            # "NUTS",
```

Comment:
Commented out because NUTS is failing to compute samples. Issue opened to address the issue: https://github.com/VTDA-Group/superphot-plus/issues/163.